lib/builders/encounter-query.js

```diff
@@ -14,9 +18,14 @@ function buildEncounterQuery (parameters, resourceType = 'Encounter') {
   }
 
   if (parameters.class) {
-    query.class = parameters.class
+    const tokens = parameters.class.split(',')
+    if (tokens.length === 1) {
+      query.class = tokens[0]
+    } else {
+      const filters = tokens.map(buildClassFilter)
+      query.$or = (query.$or || []).concat(filters)
```

Comment:
@tumbledwyer This `$or` should be nested within a `$and` like:

https://github.com/bausmeier/fhir-store/blob/8ec1a32cb6cb49f04b9c0f46a6d26d0bd7415408/lib/builders/practitioner-query.js#L21

If you don't do this then it would be combined with the existing or cases which doesn't make sense. For example:

```javascript
{
  $or: [
    {id: '1'},
    {id: '2'},
    {class: 'outpatient'},
    {class: 'ambulatory'}
  ]
}
```

This would return encounters with id '1' or id '2' *or* class 'outpatient' or class 'ambulatory', instead of encounters with id '1' or id '2' *and* class 'outpatient' or class 'ambulatory'